lib/UnoCore/Targets/Android/com/fuse/App.java

```diff
@@ -134,7 +134,6 @@ public void onResume() {
         ActivityNativeEntryPoints.cppOnResume();
         com.fuse.Activity.onResume();
         focusOnResume();
-
```

Comment:
This looks like an unneeded change. Please drop it.

Reply:
Haha, I knew you would say that

Reply:
Sorted.